lib/widgets/init.lua

```diff
@@ -30,8 +28,23 @@ return function(Iris: Types.Internal)
         end
     end
 
+    function widgets.setGuiInset()
+        local inset = widgets.GuiService:GetGuiInset()
+
+        -- Inset is not guaranteed to be set upon initialization.
+        if inset.Magnitude > 0 then
+            widgets.GuiInset = inset
+        end
+
+        return inset
+    end
+
+    function widgets.getGuiInset()
+        return widgets.GuiInset or widgets.setGuiInset()
```

Comment:
if `widgets.GuiInset` is `0` then it will fetch the inset every time rather than caching it. Is it worth changing the check to be:
```lua
return widgets.GuiInset ~= nil and widgets.GuiInset or widgets.setGuiInset()
```
Just in case future updates change this behaviour?